### PR TITLE
Map method_7288 to findRespawnPosition

### PR DESCRIPTION
**Pull request by haykam821**

### CODE REVIEW

**Comment by liach** [comment]
Mind name the parameters as well?

**Comment by modmuss50** [comment]
Looks fine, parameter names would be nice. If not I dont see a reason to not merge it. A name is better than no name.

**Review by liach — APPROVED** [review]

**Comment by liach** [comment]
Feel free to change base branch if you want to merge this into 1.15.2 snapshot instead of 1.15.1.

File: mappings/net/minecraft/entity/player/PlayerEntity.mapping [hunk 1]
```diff
@@ -107,6 +107,9 @@ CLASS net/minecraft/class_1657 net/minecraft/entity/player/PlayerEntity
 		ARG 1 entity
 		ARG 2 hand
 	METHOD method_7288 findRespawnPosition (Lnet/minecraft/class_4538;Lnet/minecraft/class_2338;Z)Ljava/util/Optional;
+		ARG 0 worldView
```
Comment (Juuxel) [inline mappings/net/minecraft/entity/player/PlayerEntity.mapping]: This should be either `world` or `view`. (Which one did we decide to use for `WorldView`, or is it mixed?)

**Review by Juuxel — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/entity/player/PlayerEntity.mapping (hunk 1 above)
Comment (liach) [reply mappings/net/minecraft/entity/player/PlayerEntity.mapping]: preferably `world`. `view` is more used for `BlockView` etc.

Comment (liach) [reply mappings/net/minecraft/entity/player/PlayerEntity.mapping]:
```suggestion
		ARG 0 world
```

File: mappings/net/minecraft/entity/player/PlayerEntity.mapping [hunk 2]
```diff
@@ -106,6 +106,10 @@ CLASS net/minecraft/class_1657 net/minecraft/entity/player/PlayerEntity
 	METHOD method_7287 interact (Lnet/minecraft/class_1297;Lnet/minecraft/class_1268;)Lnet/minecraft/class_1269;
 		ARG 1 entity
 		ARG 2 hand
+	METHOD method_7288 findRespawnPosition (Lnet/minecraft/class_4538;Lnet/minecraft/class_2338;Z)Ljava/util/Optional;
+		ARG 0 worldView
+		ARG 1 spawnBlockPos
```
Comment (liach) [inline mappings/net/minecraft/entity/player/PlayerEntity.mapping]:
```suggestion
		ARG 1 spawnPos
```

**Review by liach — APPROVED** [review]

**Review by Juuxel — APPROVED** [review]

**Comment by liach** [comment]
Note: base branch should change from 1.15.1 to 1.15.2 pre 2

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**